File: data/sample/atcoder/abc101_a/Go/impl_03.go
package main

import (
	"fmt"
)
 
func main() {
	var s string
	var i int
	fmt.Scanf("%s", &s)
	for _, c := range s {
		if (c == '+') {
		  i++
		}
		if (c == '-') {
		  i--
		}
	}
	fmt.Printf("%d", i)
}

File: data/sample/atcoder/abc101_a/C/impl_03.c
#include<stdio.h>
int main()
{
int i;
char S[5];
int count=0;
scanf("%s",S);
for(i=0;i<4;i++)
{
if(S[i]=='+')

count++;

else

count--;

}
printf("%d\n",count);
return 0;
}
